### DISCUSSION TITLE
Dremio ClickHouse ARP Connector

### DISCUSSION THREAD

**Discussion by mmsmdali** [ClickHouse/ClickHouse#36708] [Show and tell]
[Dremio ClickHouse ARP Connector](https://github.com/mmsmdali/dremio-clickhouse-arp-connector) allows Dremio to connect to and query data in ClickHouse Columnar Database.

**Comment by gk1089** [comment]
Hello Ali ji!
Great to see that you finally got it implemented! Does it implement all the query commands of Clickhouse? 

Also, have you tested it for performance: through Dremio vs. through native Clickhouse queries? A short benchmarking post would be very helpful. And lastly, are there any limitations to this connector?

I am going to try it out and get back to you in case of any problems.

Thanks again.

**Comment by mmsmdali** [comment]
https://github.com/mmsmdali/dremio-clickhouse-arp-connector/issues/5
